### PR TITLE
Fixes #7189 - Updates Wait-Job

### LINKED ISSUE #7189: Rephrase the statements focused on the "command prompt", generalize to "jobs" and "completed" to "terminated"
`Wait-Job` is about _synchronous waiting_, irrespective of whether it is invoked from the interactive prompt or from within a script - the point is that _execution is blocked_ until the waited-for condition is met.

E.g., instead of 

> Suppresses the command prompt until one or all of the PowerShell background jobs running in the session are completed.

I suggest saying something like:

> Waits until any or all of the specified PowerShell jobs have terminated.

Note:

* It isn't just about _background_ jobs - the local, child-process-based form of jobs - but also about _remote_ jobs and _thread jobs_, so just "jobs" seems like the better abstraction.

* It isn't just about _completion_, it's about _termination for whatever / a specific set of reasons_ - as already discussed in the description of the `-Force` parameter, though it would make sense to at least give a brief summary in the "Description" section.
 
* It isn't about _all_ jobs, it's about all jobs _specified_, given that `Wait-Job` requires the jobs to operate on as explicit input.

Further references to "command prompt", "background job", and "completed" should be adjusted accordingly.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f4089f31-6016-1428-3428-fdd2071acc8e
* Version Independent ID: 9034a6e7-2846-023e-227a-2b10c124ad15
* Content: [Wait-Job (Microsoft.PowerShell.Core) - PowerShell](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/wait-job?view=powershell-7.1&WT.mc_id=ps-gethelp)
* Content Source: [reference/7.1/Microsoft.PowerShell.Core/Wait-Job.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/7.1/Microsoft.PowerShell.Core/Wait-Job.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

### PR DESCRIPTION
**Pull request by chasewilson**
# PR Summary

Updates for more accurate terminology in the `Wait-Job` docs in all versions

## PR Context

Fixes #7189
Fixes [AB#1811966](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1811966)

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Preview content
- [x] Version 7.1 content
- [x] Version 7.0 content
- [x] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Core/Wait-Job.md [hunk 1]
```diff
@@ -54,16 +53,23 @@ Wait-Job [-Any] [-Timeout <Int32>] [-Force] [-Filter] <Hashtable> [<CommonParame
 
 ## DESCRIPTION
 
-The `Wait-Job` cmdlet waits for Windows PowerShell background jobs to finish before it displays the
-command prompt. You can wait until any background job is complete, or until all background jobs are
-complete, and you can set a maximum wait time for the job.
+The `Wait-Job` cmdlet waits for a job to be in a terminating state before it displays the command
+prompt. The terminating states are:
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Core/Wait-Job.md:57]:
```suggestion
The `Wait-Job` cmdlet waits for a job to be in a terminating state before continuing execution.
The terminating states are:
```

File: reference/7.0/Microsoft.PowerShell.Core/Wait-Job.md [hunk 2]
```diff
@@ -54,16 +53,23 @@ Wait-Job [-Any] [-Timeout <Int32>] [-Force] [-Filter] <Hashtable> [<CommonParame
 
 ## DESCRIPTION
 
-The `Wait-Job` cmdlet waits for PowerShell background jobs to finish before it displays the
-command prompt. You can wait until any background job is complete, or until all background jobs are
-complete, and you can set a maximum wait time for the job.
+The `Wait-Job` cmdlet waits for a job to be in a terminating state before it displays the command
+prompt. The terminating states are:
```
Comment (sdwheeler) [inline reference/7.0/Microsoft.PowerShell.Core/Wait-Job.md:57]:
```suggestion
The `Wait-Job` cmdlet waits for a job to be in a terminating state before continuing exectution.
The terminating states are:
```

File: reference/7.1/Microsoft.PowerShell.Core/Wait-Job.md [hunk 3]
```diff
@@ -54,16 +53,23 @@ Wait-Job [-Any] [-Timeout <Int32>] [-Force] [-Filter] <Hashtable> [<CommonParame
 
 ## DESCRIPTION
 
-The `Wait-Job` cmdlet waits for PowerShell background jobs to finish before it displays the
-command prompt. You can wait until any background job is complete, or until all background jobs are
-complete, and you can set a maximum wait time for the job.
+The `Wait-Job` cmdlet waits for a job to be in a terminating state before it displays the command
+prompt. The terminating states are:
```
Comment (sdwheeler) [inline reference/7.1/Microsoft.PowerShell.Core/Wait-Job.md:57]:
```suggestion
The `Wait-Job` cmdlet waits for a job to be in a terminating state before continuing execution.
The terminating states are:
```

File: reference/7.2/Microsoft.PowerShell.Core/Wait-Job.md [hunk 4]
```diff
@@ -53,16 +52,23 @@ Wait-Job [-Any] [-Timeout <Int32>] [-Force] [-Filter] <Hashtable> [<CommonParame
 
 ## DESCRIPTION
 
-The `Wait-Job` cmdlet waits for PowerShell background jobs to finish before it displays the
-command prompt. You can wait until any background job is complete, or until all background jobs are
-complete, and you can set a maximum wait time for the job.
+The `Wait-Job` cmdlet waits for a job to be in a terminating state before it displays the command
+prompt. The terminating states are:
```
Comment (sdwheeler) [inline reference/7.2/Microsoft.PowerShell.Core/Wait-Job.md:56]:
```suggestion
The `Wait-Job` cmdlet waits for a job to be in a terminating state before continuing execution.
The terminating states are:
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
See the suggestions

File: reference/5.1/Microsoft.PowerShell.Core/Wait-Job.md (hunk 1 above)
Comment (chasewilson) [reply reference/5.1/Microsoft.PowerShell.Core/Wait-Job.md:57]: My vscode crashed right after our conversation and lost this change for some reason. this was originally corrected. Thanks for the check.

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**